great_expectations/util.py

```diff
@@ -1656,10 +1663,90 @@ def convert_ndarray_decimal_to_float_dtype(data: np.ndarray) -> np.ndarray:
     return convert_decimal_to_float_vectorized(data)
 
 
-def get_context():
-    from great_expectations.data_context.data_context import DataContext
+def get_context(
+    project_config: Optional[Union["DataContextConfig", dict]] = None,
+    context_root_dir: Optional[str] = None,
+    runtime_environment: Optional[dict] = None,
+    ge_cloud_base_url: Optional[str] = None,
+    ge_cloud_access_token: Optional[str] = None,
+    ge_cloud_organization_id: Optional[str] = None,
+    ge_cloud_mode: Optional[bool] = False,
```

Comment:
```suggestion
    ge_cloud_mode: bool = False,
```
Shouldn't the type here just be `bool`?